tests/modules/withsubdomain/withsubdomain.py

```diff
@@ -1,8 +1,8 @@
-
 from dataclasses import dataclass
 
 # the import of the horsepower_to_kilowatt function should not break the parsing
```

Comment:
I think either the comment needs updating or there is a missing import.

Reply:
it is a function defined in the imported `tests.modules.withsubdomain.subdomain.insubdomain` module

See https://github.com/lucsorel/py2puml/blob/438ddcb4b3b2c7dd7cc261fd05517dfecceca9bb/tests/modules/withsubdomain/subdomain/insubdomain.py

It seems that the comment needs rephrasing.

TODO :pray: